src/Grassmann.jl

```diff
@@ -231,6 +231,10 @@ function generate_algebra(m,t,d=nothing,c=nothing)
     !isnothing(d) && generate_derivation(m,t,d,c)
 end
 
+Base.map(fn, x::MultiVector{V}) where V = MultiVector{V}(map(fn, value(x)))
+Base.map(fn, x::Chain{V,G}) where {V,G} = Chain{V,G}(map(fn,value(x)))
+Base.map(fn, x::Simplex{V,G,B}) where {V,G,B} = fn(value(x))*B
```

Comment:
It could be nice in the future to also support sparse maps.

I also thought it could be cool if you could actually map over individual `Simplex` objects for a given `Chain` or `MultiVector` in some basis, returning a completely new collection of `Simplex` objects that could be in totally different subspaces as the original ones.  Or maybe some could be filtered out, etc.

Reply:
these methods will be moved to `src/composite.jl` and making it sparse should be fairly easy, one simply has to skip over the null values by checking them with `Grassmann.isnull` to decide whether it needs to apply

Reply:
these methods will be moved to `src/composite.jl` and making it sparse should be fairly easy, one simply has to skip over the null values by checking them with `Grassmann.isnull` to decide whether it needs to apply